Assets/Scripts/test.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.AI;
+
+public class enemyMovement : MonoBehaviour
```

Comment:
seems like a test class you forgot to remove

Reply:
otherwise, `enemyMovement` -> `EnemyMovement`

Reply:
and the class name does not correspond to the file name. It prevents from actually attaching the component